N/nativefiledialog_extended/build_tarballs.jl

```diff
@@ -0,0 +1,55 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "nativefiledialog_extended"
+version = v"1.1.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/btzy/nativefiledialog-extended/", "5786fabceeaee4d892f3c7a16b243796244cdddc", "nativefiledialog-extended")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd nativefiledialog-extended/
+NFD_APPEND_EXTENSION=OFF
+if [[ "${target}" == *-linux-* ]] ; then
+    NFD_PORTAL=ON
```

Comment:
You dropped the default value `NFD_PORTAL=OFF` above, is that intentional?

BTW I used "OFF" as default value for both options in my example but of course I have no idea about the code and whether this is sensible / correct, it is on you to decided that :-)

Reply:
Not intentional, thanks for the notice.

I decided to use `OFF`. It is the better default, as it only makes sense on Linux.